app/assets/stylesheets/modules/data_access.css.scss

```diff
@@ -192,31 +192,151 @@ section.data-access-next {
 
 .data-access-collections-list {
   list-style-type: disc;
-  padding-left: 2em;
+  margin-left: 2em;
+}
+
+.data-access-collections-list-panel {
+
+  list-style-type: none;
+  margin: 0;
+
+  .data-access-collection {
+    border: 1px solid #bbb;
+    border-left: 5px solid #34495e;
+    border-radius: 3px;
+    padding: 1em;
+
+    &.closed, &.complete {
+      border-left: 5px solid #27ae60;
+    }
+
+    &.failed, &.canceled, &.cancelling, &.submit_rejected, &.submit_failed, &.quote_rejected, &.quote_failed, &.not_validated {
+      border-left: 5px solid #e74c3c;
+    }
+  }
 }
 
 .data-access-collection {
   margin: 2em 0;
 }
 
 .data-access-collection-name {
+  display: inline-block;
   font-size:1.2em;
   font-weight: bold;
-  margin: .5em 0;
+  margin: 0;
+}
+
+.data-access-collection-body {
+  margin-top: 0.5em;
+  display: flex;
+  align-items: center;
+}
+
+.data-access-collection-state {
+  width: 15%;
+  margin-top: 0.16em;
+  padding: .55em;
+  background: #34495e;
+  border-radius: 2px;
+  vertical-align: top;
+  color: #fff;
+  text-align: center;
+  text-transform: capitalize;
+
+  &.closed, &.complete {
+    background: #27ae60;
+  }
+
+  &.failed, &.canceled, &.cancelling, &.submit_rejected, &.submit_failed, &.quote_rejected, &.quote_failed, &.not_validated {
+    background: #e74c3c;
+  }
+}
+
+.data-access-collection-state-text {
+  width: 83%;
+  margin-left: 1.5%;
+
+  p {
+    margin: 0;
+  }
 }
 
 .data-access-collection-actions {
-  margin: .5em 0;
+  margin: .75em 0 0;
```

Comment:
Small comment here, and I realize the previous value didn't do this, but let's try to not start values with `.`. Instead, always lead with a 0. So, `0.75em`.